docs/projects/README.md

```diff
@@ -0,0 +1,470 @@
+of the project during planning, then something potentially big was missed during
+planning.
+
+In these cases, it is worth taking a step back to ensure that the full scope of
+the plan is defined and being followed.
+
+#### Notes for Implementers
+
+With sufficient planning, it may make certain issues able to be easily
+implemented at the same time, in a single PR. However, this should be avoided.
+The implementation plan will have split the work into small, atomic, and
+digestible chunks of work that progress the project at an appropriate pace. Keep
+in mind that small PRs are reviewed _much_ faster than bigger PRs. While
+implementing two or three issues in one go may _feel_ faster, it is often slower
+and riskier, due to increased review overhead, than going step-by-step.
+
+#### Notes for code reviewers
+
+Code reviewers should focus primarily on the technical aspects of the
+implementation required by the issue. At times, during the course of a project,
+unforeseen amiguities will discovered during implementation and sometimes an
+alarm bell must be rung. However, these should be reserved for issues that will
+cause long-term harm to the project. Merely disagreeing with a particular detail
+of the project's implementation details is not sufficient for holding up a PR
+that is part of a larger, carefully planned project, for which a technical
+implementation plan has already been reviewed and approved. Therefore, avoid
+raising objections to the broad technical decisions of a PR unless they do cause
+long term harm to the project. This allows projects to move along at an
+expedient pace and trusts that the project planners did their due diligence in
+considering alternatives.
+
+Of course, exceptions to this do exist and reviewers are expected to point out
+when there is a significant issue with the implementation details of a PR that
+will cause long-term harm.
+
+Reviewers should also be mindful to point out when an implementer has
+implemented multiple issues in a single PR, in a manner which makes reviewing
+difficult or much more time-consuming.
+
+### Delivery (`status: Shipped`)
+
+After shipping, the project enters a period of evaluation. We do this to meaure
+the success of the project and determine if it achieved the desired goal, but
+also to reflect on and make improvements to our own team processes.
+
+The success criteria for this phase are determined and refined in earlier stages
+of the project. At this point we should have a set of measurable criteria by
+which to evaluate the project. This might be things like analytics events around
+usage of a feature; a reduction in or the disappearance of a production bug; or
+changes in traffic to Openverse, as examples.
+
+During this phase, the project lead should report updates on these metrics
+regularly to help the team make a determination about changing the project
+status to `Success` or `Rollback`. Please make these updates on the project
```

Comment:
This line mentions `Rollback` and the status list at the bottom of the doc mentions `Rollback`, but the next section says `status: Reverted` - they seem to describe the same thing so which would be the preferred term to use?